packages/core/src/components/Modal/Modal.tsx

```diff
@@ -99,6 +99,7 @@ const Modal: FC<ModalProps> & { width?: typeof ModalWidthEnum } = ({
   closeButtonAriaLabel = "Close",
   contentSpacing = false,
   zIndex = 10000,
+  unmountOnClose = true,
```

Comment:
I can't comment on line 85 but maybe we need to change the comment explaining about this prop